tslint.json

```diff
@@ -1,83 +1,11 @@
-// From https://github.com/palantir/tslint/blob/master/docs/sample.tslint.json
 {
+  "extends": "@salesforce/dev-config/tslint",
   "rules": {
-    "class-name": true,
-    "curly": true,
-    "forin": true,
-    "jsdoc-format": true,
-    "label-position": true,
-    "member-access": true,
-    "new-parens": true,
+    "no-reserved-keywords": false,
     "no-any": false,
```

Comment:
This one also created too much noise. We should be taken care of by our strict rules in tslint.json